_posts/history/2022-02-06-a-brief-history-of-usrse.md

```diff
@@ -0,0 +1,372 @@
+
+Early members agreed to focus on the _engineers_, which is reflected in "community" being the first part of the [US-RSE mission](https://us-rse.org/about/mission/).  
+
+As the website content developed, Charles Ferenbaugh created a handful of candidates for an official US-RSE logo [[ref](https://docs.google.com/presentation/d/13d63A3gNEijKp4v2Iw6pwumJSBbJRSAh/edit?usp=sharing&ouid=105944557292136771119&rtpof=true&sd=true)] and a vote within slack chose the winner.
+Chris Hill purchased the us-rse.org domain name, and the new site with the new logo was published [[ref](https://github.com/USRSE/logo/blob/2652561bc29df96bf469e4f44fb86073cc388d47/current_logo/main_logo_transparent.png)].  
+
+
+![first US-RSE logo]({{ site.baseurl }}/assets/posts/history/2022-02-brief-history-usrse/image7.png "The first logo of US-RSE created in 2019.")
+
+
+### Promoting US-RSE
+
+The horses were off to publicize US-RSE! In early 2019, a Google Group and Drive were created, and a handful of members used the drive to collaborate on [a document describing the mission for the new organization](https://www.google.com/url?q=https://docs.google.com/document/d/1q_C86-6UCUY60A5KqY4IeE4M_kwPC5eOZYBkd_5M8nU/edit&sa=D&source=docs&ust=1644181697879236&usg=AOvVaw1UiXieggrluVsq8_ujfO4-).
+In April 2019 Sandra Gesing created a mailing list (via the Google Group) to promote the association, and Christina Maimone created the Twitter handle us_rse and membership form in June of that same year [[ref](https://twitter.com/us_rse)].
+Indeed, it cannot be overlooked that the success of an effort or organization is hugely influenced by the ability of that organization to reach others. Reaching others can be on a small scale such as giving talks, but more public and easy to see venues like social media also play a role. 
+
+
+### Leadership
+
+Ian Cosden continued to take the lead in pushing US-RSE forward to a more formal organization.
+An email from May 14, 2019 established a steering committee for US-RSE:
+
+
+> Dear all [Jay Jay Billings, Jeff Carver, Charles Ferenbaugh, Sandra Gesing, Chris Hill, Dan Katz, Christina Maimone, Lance Parsons, Jordan Perr-Sauer],
+>
+> In the last couple months the US-RSE group has clearly seen a rise in interest, membership, and engagement.
+> The growth is extremely encouraging, and with a small amount of effort, it’s only a matter of time before we reach a significant level of activity. 
+>
+> I wanted to reach out, not via slack, but in a more composed email.
+> The nine of you (plus me) are all listed on the website as the current steering committee, for the sole reason that you either (a) wrote your name as a contributor in the document that helped define our mission or (b) expressed interest in participating around the same time.
+> When I put the new website together, I just ran with it and appointed us the "Steering Committee."
+For better or worse, that’s all it took in December. 
+
```

Comment:
```suggestion
>
```

Reply:
to keep the quote as a single quote, rather than the two it displays as now

Reply:
ah thanks! Looks like I just forgot one :)